development/Dockerfile

```diff
@@ -6,11 +6,11 @@
 # -------------------------------------------------------------------------------------
 # !!! USE CAUTION WHEN MODIFYING LINES BELOW
 
-# Accepts a desired Nautobot version as build argument, default to 2.0.0
-ARG NAUTOBOT_VER="2.0.0"
```

Comment:
I don't think there's a need to update this at all. I'd leave it at the lower values since it gets overridden by invoke and CI all the time.